app/views/catalog/availability/_badges.html.erb

```diff
@@ -11,8 +11,18 @@
       </span>
     <% end %>
   <% end %>
-  <%= tag('br') if availability_present?(doc_avail_values, document) %>
-  <%= render_physical_avail_spans(doc_avail_values, document.id) %>
+  <% if doc_avail_values[:physical_holdings].empty? && document.bound_with? %>
+    <%= tag('br') if doc_avail_values[:online_available] || document.url_fulltext.present? %>
+    <%= phys_label_span('Check Holdings') %>
+    <%= tag.span(
+          tag.a('LOCATE/REQUEST', href: service_page_link, target: '_blank',
```

Comment:
Can we add `rel: 'noopener noreferrer'` to the link? We add that to links across the app, probably to prevent this [vulnerability](https://web.dev/external-anchors-use-rel-noopener/)

Reply:
Commit below.